### PR TITLE
Add GitHub Actions for lints and tests

### PR DESCRIPTION
**Pull request by reweeden**
isort and flake8 can (and should) be configured using a `setup.cfg`. Still need to see about a coverage reporting tool.

Once we get this working how we like we can refactor to a reusable workflow.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: lint
+on:
+  pull_request:
```
Comment (reweeden) [inline .github/workflows/lint.yml:3]: Maybe run on branches too?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r lambda/requirements.txt pytest
+
+    - run: pip install pytest-github-actions-annotate-failures
+
+    - run: pytest --cov=lambda --cov-report=term-missing --cov-report=xml --cov-branch tests
+
+    # TODO(reweeden): Report coverage
```
Comment (reweeden) [inline .github/workflows/test.yml:28]: Codecov is free for open source. We could use it on this repo.

Comment (mckadesorensen) [reply .github/workflows/test.yml:28]: That sounds good to me.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name: test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - devel
+      - master
```
Comment (Jlrine2) [inline .github/workflows/test.yml:8]: any particular reason not to run on push to any branch?

**Review by Jlrine2 — APPROVED** [review]

File: .github/workflows/test.yml (hunk 3 above)
Comment (reweeden) [reply .github/workflows/test.yml:8]: Just so it doesn't run twice on PRs.

**Pull request merged**